testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -660,6 +679,65 @@ impl ClusterSwarmKube {
         )
         .await
     }
+
+    async fn put_file(
+        &self,
+        node: &str,
+        pod_name: &str,
+        path: &str,
+        content: Vec<u8>,
+    ) -> Result<()> {
+        let bucket = "toro-cluster-test-flamegraphs";
+        let run_id = env::var("RUN_ID").expect("RUN_ID is not set.");
+        self.s3_client
+            .put_object(PutObjectRequest {
+                bucket: bucket.to_string(),
+                key: format!("data/{}/{}/{}", run_id, pod_name, path),
```

Comment:
One more ask - do you mind updating path for flamegraphs, to use similar name like `/flamegraphs/{run_id}-libra-node-perf.svg`
E.g. (a) move it to flamegraphs section, and (b) user run-id instead of random id

Reply:
Sure.

Reply:
https://toro-cluster-test-flamegraphs.s3-us-west-2.amazonaws.com/flamegraphs/ct-0-cluster-test-changyonghu-1596236068-libra-node-perf.svg